packages/@ember/debug/lib/deprecate.ts

```diff
@@ -170,13 +184,15 @@ if (DEBUG) {
     @param {String} message A description of the deprecation.
     @param {Boolean} test A boolean. If falsy, the deprecation will be displayed.
     @param {Object} options
-    @param {String} options.id A unique id for this deprecation. The id can be
+    @param {String} [options.id] A unique id for this deprecation. The id can be
```

Comment:
I think the brackets here infer that the value is optional, see http://yui.github.io/yuidoc/syntax/index.html#param.

```suggestion
    @param {String} options.id A unique id for this deprecation. The id can be
```